common/src/test/java/com/revenuecat/purchases/common/OfferingsTest.kt

```diff
@@ -129,7 +124,7 @@ class OfferingsTest {
     @Test
     fun `No offerings`() {
         val offerings =
-            JSONObject("{'offerings': [], 'current_offering_id': null}").createOfferings(emptyMap())
+            JSONObject("{'offerings': [], 'current_offering_id': ''}").createOfferings(emptyMap())
```

Comment:
This test is what made me reconsider this approach... Looks like `JSONObject.getString` in the android sdk with a null value would work. However, that same method on the official json library would throw an exception saying that the value is not a string... When considering that difference, we could potentially have tests that differ in behavior from their production code... So I considered this approach is problematic